src/figuresClasses.ts

```diff
@@ -1,11 +1,91 @@
-export interface Figure {}
+/* eslint-disable @typescript-eslint/lines-between-class-members */
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: string;
+  color: 'red' | 'green' | 'blue';
```

Comment:
u always make duplicates. Move it to a separate `union type' - `Color'
```suggestion
  color: Color;
```